projects/distributed-tracing/src/shared/dashboard/data/graphql/waterfall/trace-waterfall-data-source.model.ts

```diff
@@ -34,10 +35,18 @@ export class TraceWaterfallDataSourceModel extends GraphQlDataSourceModel<Waterf
   })
   public entrySpanId?: string;
 
+  @ModelProperty({
+    key: 'start-time',
+    required: true, // Todo
```

Comment:
What's the todo here?

Reply:
oh sorry. I need to set required: false. Kept it to true so that I don't miss setting it in a dashboard json. Will update